gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/FormsResourceTest.java

```diff
@@ -59,6 +58,7 @@ public void shouldGetForm() throws IOException {
         mockForm.setReferenceId(domainId);
 
         doReturn(Maybe.just(mockForm)).when(formService).findByDomainAndTemplate(domainId, Template.LOGIN.template());
+        doReturn(Single.just(mockForm)).when(formService).getDefaultByDomainAndTemplate(domainId, Template.LOGIN.template());
```

Comment:
There is a missing TestCase, here `findByDomainAndTemplate` return a value so the result of `getDefaultByDomainAndTemplate` isn't used to generate the response.

We should have the same test but with `findByDomainAndTemplate` that return `Maybe.emtpy`

Reply:
new test added as suggested